task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -38,7 +38,7 @@ spec:
         . /utils.sh
         SNYK_TOKEN="$(cat /etc/secrets/snyk_token)"
         if [[ -z $SNYK_TOKEN ]]; then
-          echo "SNYK_TOKEN is empty and please set params.SHARED_SECRET" | tee stdout.txt
+          echo "SNYK_TOKEN is not defined, please create the $(params.SNYK_SECRET) secret containing the Snyk token." | tee stdout.txt
```

Comment:
shall we explicitly mention that secret key must be `snyk_token` ?

Reply:
Good point, adding that.